plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchQueryBuilder.java

```diff
@@ -159,7 +159,7 @@ private static Object getValue(Type type, Object value)
             return Float.intBitsToFloat(toIntExact(((Long) value)));
         }
         if (type.equals(VARCHAR)) {
-            return ((Slice) value).toStringUtf8();
+            return ((Slice) value).length() == 0 ? 0 : ((Slice) value).toStringUtf8();
```

Comment:
I'm not sure why special handling is required only in VARCHAR types. Is it a Elasticsearch bug? Please leave code comment.

Reply:
@ebyhr I am not sure it is ES bug or not, so I create a issue for ES.
https://github.com/elastic/elasticsearch/issues/91263
After that, i will try to fix this through right way.